### ISSUE TITLE
Please provide complete example resource loading outside CreateResource for CanvasAnimatedControl

### ISSUE THREAD

**Issue by El-Wray** [Microsoft/Win2D#119]
Hi, developers!
I can not compile code example for Resource loading and CanvasAnimatedControl. Looks like method "LoadResourcesForLevelAsync" contains some mistakes. Can anybody please provide complete example that can be compiled and without mistakes?
Thanks!

**Comment by clandrew** [comment]
Hi. Could you please paste the compile error here?
That code sample does contain some placeholder elements, such as '...'
```
levelBackground = await CanvasBitmap.LoadAsync(resourceCreator, ...);
```
These aren't meant to compile, and are for example purposes only.

**Comment by El-Wray** [comment]
Hi, if you go to "Resource loading and CanvasAnimatedControl" section you can see small part of code with "LoadResourcesForLevelAsync" method. This method contains next errors:
1) "GameLoopSynchronizationContext" static class has no "RunOnGameLoopAsync" method, only "RunOnGameLoopThreadAsync".
2) When I try to use this method, visual studio show "Return type is void" error.

**Comment by clandrew** [comment]
Looks like a typo in the sample code! This method is meant to be RunOnGameLoopThreadAsync. Much appreciated, catching this. We'll update the doc. 

Does fixing this allow your code to compile?

**Comment by shawnhar** [comment]
We fixed this doc error in Win2D 1.0.0, which was released today.  Thanks for pointing it out!

Please reactivate if you have any more issues with the corrected doc.
